docs/teams/phoenix_integration.md

```diff
@@ -0,0 +1,298 @@
+
+#### Set a static cookie
+
+Set the `RELEASE_COOKIE` environment variable on your production machines to ensure a static cookie value across deployments, then restart or redeploy your app.
+
+Use the same value for your `PHOENIX_APP_COOKIE` [Livebook secret](#set-up-environment-secrets).
+
+Learn more about [setting the cookie of an Elixir release here](https://hexdocs.pm/mix/Mix.Tasks.Release.html#module-options).
+
+#### Enable long node names
+
+Livebook requires long node names. Configure the `RELEASE_DISTRIBUTION` environment variable inside your app's `rel/env.sh.eex` like this:
+
+```bash
+# rel/env.sh.eex
+
+export RELEASE_DISTRIBUTION=name
+```
+
+## Conveniences for working with remote code
+
+Livebook provides built-in tools to simplify working with remote code.
+
+### Remote execution smart cell
+
+The remote execution smart cell offers several advantages over manual `:erpc.call` functions:
+
+#### Built-in connection management
+
+Set the node name and cookie directly in the smart cell.
+
+![](images/remote-smart-cell-node-cookie-config.png)
+
+#### Code autocomplete
+
+Get autocomplete for functions from your remote Phoenix app.
+
+![](images/remote-cell-autocomplete.png)
+
+#### Multi-line execution
+
+Run multiple lines of code in the remote context.
+
+![](images/remote-cell-multiple-lines.png)
+
+#### Variable integration
+
+Reference notebook variables and assign results back to the notebook.
+
+![](images/remote-cell-and-code-cell.png)
+
+### Kino.RPC
+
+For more flexibility, use [`Kino.RPC`](https://hexdocs.pm/kino/Kino.RPC.html) directly. This is the same module the remote execution smart cell uses behind the scenes.
+
+Let's say you're building a Livebook app to show user counts by status. Given your Phoenix app has a function `MyApp.Users.count_by_status/1`, you can call it using the remote execution smart cell:
+
+![](images/remote_execution_with_argument.png)
+
+But what if you want the `status` variable to come from a form input?
+
+![](images/form_with_needed_remote_call.png)
+
+Now we need a way to call the remote function passing an argument that's coming from the form. To do that, we can extract the remote function call into a reusable module.
+
+First, convert your remote execution smart cell to a regular code cell by clicking in the pencil icon:
+
+![](images/convert_remote_executio_smart_cell.png)
+
+You'll see that the smart cell was generating this code:
+
+```elixir
+require Kino.RPC
+node = my_app_node
+Node.set_cookie(node, my_app_cookie)
+Kino.RPC.eval_string(node, ~S"MyApp.Users.count_by_status(status)", file: __ENV__.file)
+```
+
+Extract this into a module so you can pass the `status` value as an argument:
+
+```elixir
+Node.set_cookie(my_app_node, my_app_cookie)
+
+defmodule MyAppRPC.Users do
+  require Kino.RPC
+
+  def count_by_status(node, status) do
+    Kino.RPC.eval_string(node, ~S"MyApp.Users.count_by_status(status)", file: __ENV__.file)
```

Comment:
I am not sure there is a benefit in using `Kino.RPC` over `:erpc`? Perhaps we should stick with `:erpc` instead of eval? I think it is nice to show how the smart cell works but I don't think we need to discuss its internals (Kino.RPC).

Reply:
Ok, so ignore me, kino.rpc is the way to go, but it would be better if we use smart cells instead of low level API as much as possible!

Reply:
Yes, we can mention Kino.RPC and eval at the end as a protip. :)